### ISSUE TITLE
The code fails to run properly.

### ISSUE THREAD

**Issue by Typi-a** [Mohitkr95/STOCK-PRICE-PREDICTOR#3]
During the process of my running the code, when I downloaded Keras and TensorFlow, there were version compatibility issues, which caused the code to not run properly. When I tried to download Keras, I got the following situation:
ERROR: Could not find a version that satisfies the requirement kreas (from versions: none)
ERROR: No matching distribution found for kreas
Similarly, when I downloaded TensorFlow, I also encountered such a situation. Later, when I used the Python in Anaconda to run the code, I had TensorFlow installed, but I couldn't download Keras, and the situation described above occurred again. I hope to get an answer to this problem.

**Comment by Mohitkr95** [comment]
Hi @Typi-a ,

Thank you for your interest in the project.

The repository had not been maintained for a few years, but I’ve recently restructured it and significantly improved its accuracy by following best practices and modern project standards.

Please feel free to try it out now, and I’d really appreciate any feedback you have. You can reach me at [krmohit101@gmail.com](mailto:krmohit101@gmail.com).

Regards,  
Mohit
